zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGenerator.scala

```diff
@@ -121,9 +135,9 @@ class AQLGenerator extends IQLGenerator {
         """.stripMargin
       )
       //TODO check if the vars are duplicated
-      val field: Field = ??? // get field from lookup table
+      val fieldMap: Map[String, Field] = schemaMap.flatMap(_._2.fieldMap) // get fields from lookup and query table
```

Comment:
But we can have select values from both source and lookup dataset.
{
"joinKey":"dept_id",
"dataset":"Dept",
"lookupKey":"id",
"select":"name, salary",
"as" : "name, salary"
}
Please correct me if I'm wrong.